tests/socket/socket_tests.cpp

```diff
@@ -359,7 +360,7 @@ helper_functions_validation_test(
     bpf_map* connection_count_map = bpf_object__find_map_by_name(object, "connection_count_map");
     SAFE_REQUIRE(connection_count_map != nullptr);
 
-    // Attach the auth connect program at the appropriate AUTH_CONNECT layer.
+    // Attach the authorization connect program at the appropriate AUTH_CONNECT layer.
     bpf_attach_type auth_connect_attach_type =
```

Comment:
I see you changed "auth" in comments (thanks), but not in variable names like this line or the attach type define name.
Should it be changed everywhere?